CommandCenter.m

```diff
@@ -69,19 +69,27 @@ function CommandCenter_OpeningFcn(hObject, eventdata, handles, varargin)
 key = 'ROYZNcVBgWkT8xiwcg5m2Nn9Gb4EAegF2XEN1i5adWD';  % CC key (helps avoid spam)
 [path,~,~] = fileparts(mfilename('fullpath'));
 % Check for lock file
-if exist(fullfile(path,'.lock'),'file')
-    delete(hObject);
-    error(['Found a lock file, are you sure there is no other CommandCenter running?\n',...
-        'If not, you may remove this file and retry launching:\n%s'],...
-        fullfile(path,'.lock'));
-else
-    fclose(fopen(fullfile(path,'.lock'), 'w'));
+if exist(fullfile(path,'.lock'),'file')     % Is this nnecessary with the hObject.Visible check?
+    lock = fullfile(path,'.lock');
+    
+    answer = questdlg(sprintf(['Found a lock file, are you sure there is no other CommandCenter instance running?\n',...
```

Comment:
If you are allowing the removal of it via questdlg, you can modify the contents of the text that get printed to not suggest manually deleting.
My only concern for doing it this way is it is an easy way to bypass the lock mechanism, whereas before would be much more difficult for a general user - which is actually kind of good!